charts/zeppelin/templates/interpreter/role.yaml

```diff
@@ -0,0 +1,47 @@
+{{- /*
+Licensed to the Apache Software Foundation (ASF) under one or more
+contributor license agreements.  See the NOTICE file distributed with
+this work for additional information regarding copyright ownership.
+The ASF licenses this file to You under the Apache License, Version 2.0
+(the "License"); you may not use this file except in compliance with
+the License.  You may obtain a copy of the License at
+
+   http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/ -}}
+
+apiVersion: rbac.authorization.k8s.io/v1
```

Comment:
Please note that Role, ServiceAccount and Role binding are only required for the Spark interpreter. How is this implemented?

Reply:
This logic is implemented in `interpreter-spec.yaml` file, we will not mount the service account token except for the interpreter group name is `spark`.